Lib/test/libregrtest/main.py

```diff
@@ -533,6 +534,13 @@ def main(self, tests=None, **kwargs):
     def _main(self, tests, kwargs):
         self.ns = self.parse_args(kwargs)
 
+        self.getloadavg = None
+        if hasattr(os, 'getloadavg'):
+            self.getloadavg = lambda: os.getloadavg()[0]
+        elif sys.platform == 'win32' and (self.ns.slaveargs is None):
+            load_tracker = WindowsLoadTracker()
+            self.getloadavg = load_tracker.getloadavg
```

Comment:
I seems like you spawn a subprocess even in test worker processes, whereas you wrote that it's useless. This code should be moved after handling slaveargs, no? Maybe move the slaveargs handling code earlier?

Reply:
Not sure I follow, we already know if the runner is a slave based on the parsing of the arguments. The arguments have been parsed by this point, so this should be fine. I also tested and under task manager only one typeperf instance shows up under the main python process unlike before.